### ISSUE TITLE
Unable to initiate user with multi-user image

### ISSUE THREAD

**Issue by taophp** [fpiesche/docker-mailpile#10]
I tried to make mailpile-multiuser work on a Raspberry PI 3 with Raspbian.
Here is my `docker-compose.yml`:
```yaml
version: '3'

volumes:
  mailpile-steph:
  mailpile-christel:

services:
  mailpile:
    image: florianpiesche/mailpile-multiuser
    restart: unless-stopped
    volumes:
      - mailpile-steph:/home/pirate/Data/mailpile/steph
      - mailpile-christel:/home/pirate/Data/mailpile/christel
    ports:
      - 10081:80
    environment:
      MAILPILE_USERS: "steph christel"
```
 Users defined in the `MAILPILE_USERS` does not exists locally, but the `/home/pirate/Data/mailpile/steph` directory exists, with all right for everybody.

I tried to login as steph. So a script tried to initiate the user of mailpile. Below, the output:
```
This is a script to launch Mailpile as a specific user.
The user must already have Mailpile configured.

Usage: mailpile-launcher.py USERNAME IDLE-QUIT-SECONDS URL

 That user has never run Mailpile. Aborting. 

Location: /mailpile/steph/
Expires: 0

==> sudo /usr/share/mailpile/multipile/mailpile-launcher.py steph 900 127.0.0.1:54511/mailpile/steph/
==[ FAILED! Exit code: 768 ]==
```

Can you help me to solve this problem ?

**Comment by taophp** [comment]
BTW, I have tried to initiate the user with `mailpile-admin.py` from the container shell:
```
root@c089650880f9:/usr/share/mailpile/multipile# ./mailpile-admin.py --user steph --start
==> sudo -iHu "steph" -- /usr/bin/python2.7 "/usr/share/mailpile/multipile/mailpile-admin.py" --start --port="48451" --host="127.0.0.1"
==[ FAILED! Exit code: 256 ]==
==> chown -R www-data:root /var/lib/mailpile/apache
==> chmod go+rwxt /var/lib/mailpile/pids
===[ SUCCESS! ]===
```
It does not change anything.

**Comment by fpiesche** [comment]
Sorry for taking so long to reply to this - I had never really tested the multiuser image, instead assuming that it would work transparently. Sadly this isn't the case and getting Mailpile's existing multi-user system working in a Container is a nontrivial effort and basically impossible without a complex reverse proxy setup on top of what's there. I've just spent yesterday trying to fix it (as the builds here have generally been broken thanks to Debian removing `armv6` support from the Buster Docker base image) and haven't been able to get it to work.

Given that Mailpile has resumed development (currently behind closed doors) and is likely to change a lot in the future, I've decided to instead just remove the multi-user setup. Sorry :( If you still want to try this your best bet is probably to run a separate container for each user.
